kernel/src/Boot/Patches/Patches672.cpp

```diff
@@ -123,39 +124,105 @@ void Mira::Boot::Patches::install_prerunPatches_672()
 	kmem = (uint8_t *)&gKernelBase[0x0010F879];
 	kmem[0] = 0xEB;
 
-	// Enable debug rif's
-	kmem = (uint8_t*)&gKernelBase[0x66AEB0];
-	kmem[0] = 0xB0;
-	kmem[1] = 0x01;
-	kmem[2] = 0xC3;
+	// second ptrace patch
+	// via DeathRGH
+	kmem = (uint8_t *)&gKernelBase[0x0010FD22];
+	kmem[0] = 0xE9;
+	kmem[1] = 0xE2;
+	kmem[2] = 0x02;
+	kmem[3] = 0x00;
+	kmem[4] = 0x00;
+
+	// setlogin patch (for autolaunch check)
+	kmem = (uint8_t *)&gKernelBase[0x0010EC1C];
+	kmem[0] = 0x48;
+	kmem[1] = 0x31;
+	kmem[2] = 0xC0;
 	kmem[3] = 0x90;
+	kmem[4] = 0x90;
 
-	// Enable debug rifs 2
-	kmem = (uint8_t*)&gKernelBase[0x66AEE0];
-	kmem[0] = 0xB0;
-	kmem[1] = 0x01;
-	kmem[2] = 0xC3;
+	// Patch to remove vm_fault: fault on nofault entry, addr %llx
```

Comment:
Why was this added? If you are faulting, don't ignore the error, fix your code. This should be removed unless there is an explicit need for it

Reply:
This has been included in the previous firmwares so I brought it forward if it is not needed it should be removed from the other firmwares as well, let me know.